src/plugin.json

```diff
@@ -73,6 +77,226 @@
       "path": "datasource/plugin.json"
     }
   ],
+  "roles": [
+    {
+      "role": {
+        "name": "Checks reader",
+        "description": "Read checks in the Synthetic Monitoring app",
+        "permissions": [
+          { "action": "plugins.app:access", "scope": "plugins:id:grafana-synthetic-monitoring-app" },
+          { "action": "grafana-synthetic-monitoring-app:read" },
+
+          { "action": "grafana-synthetic-monitoring-app.checks:read" }
+        ]
+      },
+      "grants": ["Admin", "Editor", "Viewer"]
```

Comment:
Why do we grant this to Admin and Editor? Since they get these permissions with the Checks writer? We didn't do this in k6 and now I'm wondering if we should.

Reply:
You're right, I think it's redundant. Removed it.